mscore/scoreview.cpp

```diff
@@ -2000,6 +2000,14 @@ void ScoreView::cmd(const char* s)
             score()->upDown(true, UpDownMode::DIATONIC);
       else if (cmd == "pitch-down-diatonic")
             score()->upDown(false, UpDownMode::DIATONIC);
+      else if (cmd == "pitch-up-diatonic-alterations") {
+            if (!score()->selection().isNone())
+                  score()->transpose(TransposeMode::DIATONICALLY, TransposeDirection::UP, Key::C, 1, true, true, true);
+                  }
+      else if (cmd == "pitch-down-diatonic-alterations") {
+            if (!score()->selection().isNone())
+                  score()->transpose(TransposeMode::DIATONICALLY, TransposeDirection::DOWN, Key::C, 1, true, true, true);
+                  }
```

Comment:
The right braces should be indented only once, like 

```
else if (cmd == "pitch-up-diatonic-alterations") {
      if (!score()->selection().isNone())
            score()->transpose(TransposeMode::DIATONICALLY, TransposeDirection::UP, Key::C, 1, true, true, true);
      }
```

Reply:
Another careless mistake trying to adapt to the historical banner format

Reply:
After spending sometime reading and writing MuseScore code, I became a real fan of this banner-style indentation. Now almost any other C++ code is a pain for my eyes.